packages/taquito/src/wallet/operation.ts

```diff
@@ -14,9 +14,14 @@ import { Context } from '../context';
 import { Receipt, receiptFromOperation } from './receipt';
 import { validateOperation, ValidationResult, InvalidOperationHashError } from '@taquito/utils';
 import { BlockIdentifier } from '../read-provider/interface';
+import { InvalidConfirmationCountError, ConfirmationUndefinedError } from '../error';
 
 export type OperationStatus = 'pending' | 'unknown' | OperationResultStatusEnum;
 
+/**
+ *  @category Error
+ *  @description Error that indicates a missed block when waiting for confirmation from the blockchain
```

Comment:
Missed a block when polling to retrieve new head block. This can happen if the polling interval is greater than the time between block.